quick_start_new.md

```diff
@@ -0,0 +1,34 @@
+## Setup
+Compatible with Python versions 3.8.10 and higher
+### Install dependencies
+```bash
+# Create an virtual environment 
+# (optional but recommended)
+$ python -m venv venv 
+$ source venv/bin/activate
+# install dependencies
+(venv) $ pip install -r requirements_quick_start.txt
+```
+
+## Create config file
+```bash
+python ver_cli.py create_sources_file quickstart
+python ver_cli.py add_csv quickstart quickstart <DIR>
```

Comment:
Is it possible to give an example of using the demo dataset as `<DIR>`?

Reply:
I think I explained `<DIR>` below the code block you refer to. If you put a relative path in it, e.g., `demo_dataset/`, there is an error. So you need to give the absolute path of the dataset, e.g., `/home/user/ver/demo_dataset/`. I don't provide the path in the markdown because other people's directory structures might be different from mine. Does this answer your question well? Though, I think there may be a way to enable relative path when adding csv files. I'll make the explanation in the markdown better soon.